blocks/api/raw-handling/utils.js

```diff
@@ -34,7 +34,7 @@ const inlineWrapperWhiteList = {
 const whitelist = {
 	...inlineWhitelist,
 	...inlineWrapperWhiteList,
-	img: { attributes: [ 'src', 'alt' ] },
+	img: { attributes: [ 'src', 'alt' ], classes: [ 'alignleft', 'aligncenter', 'alignright', 'alignnone' ] },
```

Comment:
Side thought: we should start to consider performance and do some measuring. Some of the predicates that consume this data could perhaps benefit from a more barebones `switch` construct, for instance.

Reply:
Could you give an example?

Reply:
Oh, I just had [this benchmark](https://jsperf.com/proxy-request-origin-matching) in the back of my head.